docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -63,6 +63,11 @@ The `terraform` block supports the following arguments:
       registry]({{site.baseurl}}/docs/getting-started/quick-start#a-note-about-using-modules-from-the-registry) for more
       information about using modules from the Terraform Registry with Terragrunt.
 
+- `include_in_copy` (attribute): When using local file path sources, this specifies a list of glob patterns (e.g.,
```

Comment:
Is this purely additive to Terragrunt's default behavior? Or does it allow you to have Terragrunt _not_ copy some items? I'm guessing by the name it's the former... But I worry that the minute we add this, users will also request the latter. I suppose we could add an `exclude_in_copy` feature in the future...

Reply:
```suggestion
- `include_in_copy` (attribute): When using local file path sources, this specifies a list of glob patterns (e.g.,
```

Is this really the proper context on this feature? It might be worth adding some context like:

When you use the `source` param in your Terragrunt config and run `terragrunt <command>`, Terragrunt will download the code specified at `source` into a scratch folder (`.terragrunt-cache`, by default), copy the code in your current working directory into the same scratch folder, and then run `terraform <command>` in that scratch folder. By default, Terragrunt excludes hidden files and folders during the copy step. This feature allows you to specify glob patterns ...

Reply:
Sorry for the delay in responding.

> Is this purely additive to Terragrunt's default behavior? Or does it allow you to have Terragrunt not copy some items? I'm guessing by the name it's the former...

Yes this is purely additive.

> But I worry that the minute we add this, users will also request the latter. I suppose we could add an exclude_in_copy feature in the future...

I'm actually not as concerned about this, primarily because there is limited use case for exclusion in the context of `terragrunt`. I think in practice, it isn't practical to rely on terragrunt to exclude files using a lookup table, and instead should rely on other means (like moving the files to a different folder above levels, or use something like `git`). I think we can wait to add `exclude_in_copy` if someone asks for it.

> Is this really the proper context on this feature?

Ah good catch about mentioning the terragrunt working dir. Updated this section! 83c60b1d370bb4bcd056a90376825a2730c4687b